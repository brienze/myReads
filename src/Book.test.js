import React from 'react';
import {shallow} from 'enzyme';
import BookApp from './Book.js';
import {getBook, generateBookList} from './util/testData';

const myBooks = generateBookList(),
  addOrMoveBook = jest.fn(),
  book = getBook();

it('renderiza corretamente', () => {
  expect(shallow(<BookApp shelf={book.shelf} addOrMoveBook={addOrMoveBook} book={book}/>)).toMatchSnapshot();
})
