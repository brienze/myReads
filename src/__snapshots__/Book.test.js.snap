// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza corretamente 1`] = `
<li>
  <div
    className="book"
  >
    <div
      className="book-top"
    >
      <div
        className="book-cover"
        style={
          Object {
            "backgroundImage": "url(undefined)",
            "height": 193,
            "width": 128,
          }
        }
      />
      <div
        className="book-shelf-changer"
      >
        <select
          onChange={[Function]}
          value="currentlyReading"
        >
          <option
            disabled={true}
            value="none"
          >
            Move to...
          </option>
          <option
            value="currentlyReading"
          >
            Currently Reading
          </option>
          <option
            value="wantToRead"
          >
            Want to Read
          </option>
          <option
            value="read"
          >
            Read
          </option>
          <option
            value="none"
          >
            None
          </option>
        </select>
      </div>
    </div>
    <div
      className="book-title"
    >
      amet
    </div>
    <div
      className="book-authors"
    >
      Bernita Rempel
    </div>
  </div>
</li>
`;
